src/main/scala/beam/router/skim/ActivitySimSkimmer.scala

```diff
@@ -224,14 +224,18 @@ class ActivitySimSkimmer @Inject() (matsimServices: MatsimServices, beamScenario
     destination: GeoUnit,
     pathType: ActivitySimPathType
   ): Option[ExcerptData] = {
-    val individualSkims = timeBin.hours.flatMap { hour =>
-      getCurrentSkimValue(ActivitySimSkimmerKey(hour, pathType, origin.id, destination.id))
-        .map(_.asInstanceOf[ActivitySimSkimmerInternal])
-    }
-    if (individualSkims.isEmpty) {
+    if (pathType == ActivitySimPathType.WALK && timeBin != ActivitySimTimeBin.EARLY_AM) {
```

Comment:
Is it possible to start generation of skims without 'EARLY_AM' time and completely miss the walk skims generation? 
If it is possible, maybe it would be better to keep a map of OD - walk skims and generate only if a value in a map is missing?

Reply:
No, this code for ActivitySim skims is always used together with `ActivitySimTimeBin.values` which lists all possible time values.